### PR TITLE
All html assignment completed

### PR DESCRIPTION
**Pull request by anjaliasade**

### CODE REVIEW

File: anjali29/contact.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        <li>
+           <span style="font-weight:bold;">Interest :</span>Interested in web designing.
+           <a href="https://en.wikipedia.org/wiki/webdesign" target="_blank">
+            <img src="images/12.jpg" alt="Image is not available" height="200px" width="200px">
+            </a>
+        </li> 
+    </ul>
+</p>
+</div>
+
```
Comment (poojakamble) [inline anjali29/contact.html:137]: remove extra space

File: anjali29/contact.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                        <li>Spend quality time with family</li>
+                        <li>Buy a new Scooty</li>
+                        <li>Want to eat pav bhaji at chowpati</li>
+                        <li>Help the needy childrens in their education</li>
+                        <li>want to learn new things</li>
+
+                  </ul>
+                </dd>
+            </dl>
+
```
Comment (poojakamble) [inline anjali29/contact.html:81]: remove extra space

File: anjali29/contact.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+ 	 <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ 	    <link rel="stylesheet" href="fonts/material-design-iconic-font/css/material-design-iconic-font.min.css"> <link rel="stylesheet" href="css/style.css"> </head> 
+      <body style=" border-radius: 5px;">
+<div  style=" position: fixed;
+    z-index: 1;
+    height:40px;
+    width: 98%;
+    background-color: #668284;
+    margin-bottom: 10px">
+  <p style="float:left;
```
Comment (poojakamble) [inline anjali29/contact.html:15]: tab and space

File: anjali29/contact.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+ <html> 
+ <meta http-equiv="content-type" content="text/html;charset=UTF-8" /> <head> 
+ 	<meta charset="utf-8"> 
+ 	<title></title>
+ 	 <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ 	    <link rel="stylesheet" href="fonts/material-design-iconic-font/css/material-design-iconic-font.min.css"> <link rel="stylesheet" href="css/style.css"> </head> 
+      <body style=" border-radius: 5px;">
+<div  style=" position: fixed;
```
Comment (poojakamble) [inline anjali29/contact.html:9]: one tab and space

File: anjali29/contact.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+            <dl style="margin-left: 15px;"> <!-- dl specifies create list-->
+                <dd>
+                  <ul><!-- ordred list-->
+                     <li>Travelling  the world</li>
+                        <li>Spend quality time with family</li>
+                        <li>Buy a new Scooty</li>
+                        <li>Want to eat pav bhaji at chowpati</li>
+                        <li>Help the needy childrens in their education</li>
+                        <li>want to learn new things</li>
+
```
Comment (poojakamble) [inline anjali29/contact.html:77]: remove extra space

File: anjali29/headhtmltitle.html [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (poojakamble) [inline anjali29/headhtmltitle.html:3]: one tab and space

File: anjali29/bio.html [hunk 7]
```diff
@@ -0,0 +1,204 @@
+
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"><html><head><META http-equiv="Content-Type" content="text/html; charset=utf-8"></head>
```
Comment (JafarPathan) [inline anjali29/bio.html:2]: add indentation

File: anjali29/bio.html [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+      <div></div>
+      <div style="float:left;margin-top:50px;height:auto;width:99%;background-color:#00ffff">
+         <h2 style="text-decoration:underline">ABOUT ME </h2>
+         <div>
+            <p>
+               I am anjali from <em><b>solapur</b></em><br>
+               I have completed my engineering from <strong>N.K.Orchid college of engg and tech</strong><br>
+            </p>
+         </div>
+         <h3 style="text-decoration:underline">FIND ME ON THE WEB</h3>
```
Comment (JafarPathan) [inline anjali29/bio.html:22]: can you use alternative html tag for underline text

File: anjali29/bio.html [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+             <h2 style=" text-decoration: underline;">Educational Qualifications</h2>
+            <table style="border: 1px dashed black;">
+                <tr style=" font-weight:bold;">
+                    <th colspan="2" style=" padding: 2px;
+                        border: 1px solid #E88741;">institution</th>
+                     <td style=" padding: 2px;
+                        border: 1px solid #E88741; ">year of passing</td>
+                    <td style=" padding: 2px;
+                        border: 1px solid #E88741;">certificate / degree </td>
+                </tr>
```
Comment (JafarPathan) [inline anjali29/bio.html:112]: use proper table tags

File: anjali29/bio.html [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+         </div>
+         <div>
+            <h2 style="text-decoration:underline">Personality skills:</h2>
+            <p>Positive attitude, hard working ,good team player</p>
+            <div>
+               <h2 style="text-decoration:underline">Personal Information:</h2>
+               <p>
+               <ul>
+                  <li>
+                     <span style="font-weight:bold">Languages Known:</span>English,marathi,hindi
```
Comment (JafarPathan) [inline anjali29/bio.html:147]: explain you use p tag

File: anjali29/bio.html [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+                     </a>
+                  </li>
+               </ul>
+               </p>
+            </div>
+            <div>
+               <div>
+                  <div>
+                     <div>
+                        <form method="post" target="_blank" onsubmit="try {return window.confirm(&quot;This form may not function properly due to certain security constraints.\nContinue?&quot;);} catch (e) {return false;}">
```
Comment (JafarPathan) [inline anjali29/bio.html:165]: remove functions use only html and css

File: anjali29/bio.html [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+                     </div>
+                  </div>
+               </div>
+            </div>
+         </div>
+         <div>
+      </div>
+      </div>
+       </div>
+</body></html>
```
Comment (JafarPathan) [inline anjali29/bio.html:204]: u use extra closing div tag

File: anjali29/htmlanswers.txt [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+Markup is what HTML do to the text inside them. They mark it as a certain type of text(like bold, italic, underline etc).
+HTML is a language as it has its own code words like any other language.
+Due to these reasons HTML is calles Markup language.
+
+
+
+4.	Describe one problem you would likely face if you included presentation (styling) information with HTML instead of separating it in CSS?
+Ans .
+
+
```
Comment (JafarPathan) [inline anjali29/htmlanswers.txt:35]: Answer...?

File: anjali29/htmlanswers.txt [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+�	improving search engine ranking
+
+
+
+6.	Why should we add meta elements for description and keywords even though they do not have any impact on the way a web page is displayed?
+
+                 Headings, paragraphs, and line breaks
+
+7.	Is there any best practice related to how often an H1 tag should ideally appear in a single web page? If so, what is the best practice?
+
```
Comment (JafarPathan) [inline anjali29/htmlanswers.txt:51]: answer.....?

File: anjali29/htmlanswers.txt [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+
+8.	Sometimes developers who do not understand best practices properly may simply use text of a larger font size to denote headings instead of using one of the H tags. While visually both may appear identical and a casual reader would not even know the difference, such a lapse might cause problems in certain scenarios. Mention at least one scenario where such an incorrect design could create a problem.
+                   
+
```
Comment (JafarPathan) [inline anjali29/htmlanswers.txt:55]: Answer....?

File: anjali29/htmlanswers.txt [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+                           Emphasizing text in HTML
+9.	Web browsers are just one of the ways of rendering HTML pages. HTML content can also be read out by speech readers instead of being displayed visually. Based on what you have read in this section, what � according to you � would be the difference in the way the following two lines of text are read out:
+1.	You need to come here <i>right now</i>.
+2.	You need to come here <em>right now</em>
+
+
+
+10.	When we link to an image from an HTML document, does the image have to be in the same directory as the HTML page or can it exist anywhere on the Internet?
+
```
Comment (JafarPathan) [inline anjali29/htmlanswers.txt:66]: Answer...?

File: anjali29/htmlanswers.txt [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+                    <span> and <div>, the generic containers
+12 . If <span> and <div> are semantically neutral (that is, they do not confer any meaning to the content they envelop), then why would you even use them? What possible benefit could we get from using these elements in our web page?
+
+
+                   Block and inline elements
+13.List two inline elements and two block elements.
+
+14.List three differences between block and inline elements.
+
```
Comment (JafarPathan) [inline anjali29/htmlanswers.txt:83]: Answer....?

File: anjali29/bio.html (hunk 8 above)
Comment (anjaliasade) [reply anjali29/bio.html:22]: no sir i m adding css style:text-dcoration:underline on heading alternative tag is <u>my underline text</u>

File: anjali29/bio.html (hunk 10 above)
Comment (anjaliasade) [reply anjali29/bio.html:147]: <p> tag is used to designate a paragraph.

**Pull request merged**